packages/db-objects-redis/lib/objects/objectsInRedisClient.js

```diff
@@ -598,6 +606,16 @@ class ObjectsInRedisClient {
             // do this before starting with async calls ;-)
             initCounter++;
 
+            try {
+                // check if we are allowed to use sets
+                this.useSets = !!parseInt(await this.client.get(`${this.metaNamespace}objects.features.useSets`));
+            } catch (e) {
+                // if unsupported we have a legacy host
+                if (!e.message.includes('UNSUPPORTED')) {
+                    throw e;
+                }
+            }
```

Comment:
else useSets = false

??

Reply:
else its just falsy, but making it eplicitly makes sense